test/test.js

```diff
@@ -19083,6 +19083,29 @@
 
   /*--------------------------------------------------------------------------*/
 
+  QUnit.module('lodash.toSafeInteger');
+
+  (function() {
+    QUnit.test('should convert values to safe integers', function(assert) {
+      assert.expect(6);
+
+      assert.strictEqual(_.toSafeInteger(-5.6), -5);
+      assert.strictEqual(_.toSafeInteger('5.6'), 5);
+      assert.strictEqual(_.toSafeInteger(), 0);
+      assert.strictEqual(_.toSafeInteger(NaN), 0);
```

Comment:
I can say that for me this is a commonly used function. I don't think lodash is following that spec per say but like in the comments https://github.com/lodash/lodash/blob/master/lodash.js#L9720, it is loosely basing some of it's methods on that spec, why not?